sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -7508,7 +7590,7 @@
                 {
                   "label": "Are you living independently in a self-contained suite?",
                   "optionsLabelPosition": "right",
-                  "tooltip": "- The suite has a separate entrance, kitchen, bathroom and living area/bedroom; and\r\n\r\n- You are paying fair market rent and responsible for your share of the utilities (hydro, telephone, cable, etc. if not included in rent).",
+                  "tooltip": "The suite has a separate entrance, kitchen, bathroom and living area/bedroom; and\r\nYou are paying fair market rent and responsible for your share of the utilities (hydro, telephone, cable, etc. if not included in rent).",
```

Comment:
Is there a reason to "You" be in upper case since it is not the start of a new sentence?